airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml

```diff
@@ -0,0 +1,155 @@
+version: "0.1.0"
+
+definitions:
+
+  selector:
+    extractor:
+      field_pointer: [ "data" ]
+
+  requester:
+    url_base: "https://connect.mailerlite.com"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+
+
+  increment_paginator:
+    type: "DefaultPaginator"
+    url_base: "*ref(definitions.requester.url_base)"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "limit"
+    pagination_strategy:
+      type: "PageIncrement"
+      page_size: 25
+    page_token_option:
+      inject_into: "request_parameter"
+      field_name: "page"
+
+
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+
+  # STREAMS
+  # API Docs: https://developers.mailerlite.com/docs/subscribers.html
+  subscribers_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "subscribers"
+      primary_key: "id"
+      path: "/api/subscribers"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      paginator:
+        $ref: "*ref(definitions.increment_paginator)"
+
+  # API Docs: https://developers.mailerlite.com/docs/segments.html
+  segments_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "segments"
+      primary_key: "id"
+      path: "/api/segments"
```

Comment:
Why `api` doesn't belongs to the base url?

Reply:
Not sure but that's what their documentation says: https://developers.mailerlite.com/docs/#mailerlite-api

Reply:
ok, but you can simplify the path to the connectr.